src/sortStudents.ts

```diff
@@ -1,16 +1,52 @@
-
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
+
+function getAverageGrade(grades: number[]): number {
+  return grades.reduce((accum, currentValue) => {
+    return accum + currentValue;
+  }, 0) / grades.length;
+}
```

Comment:
You can write it into one line but if here is more than 80 symbols, then, better to move the method to the next line.
```suggestion
function getAverageGrade(grades: number[]): number {
  return grades
     .reduce((accum, currentValue) => accum + currentValue, 0) / grades.length;
}
```

Reply:
👍